MANIFEST.in

```diff
@@ -1,12 +1,12 @@
+include requirements*.txt
 include README.rst
 include LICENSE.txt
 include AUTHORS.txt
 include elephant/VERSION
 include elephant/current_source_density_src/README.md
 include elephant/current_source_density_src/test_data.mat
-include elephant/spade_src/LICENCE
+include elephant/spade_src/LICENSE
+recursive-include elephant/spade_src *.so *.pyd
 include elephant/test/spike_extraction_test_data.txt
 recursive-include doc *
-
-# special care for the files, used in setup.py
-include elephant/spade_src/fim_manager.py
+prune doc/_build
```

Comment:
Also, I think it needs a `recursive-include tools *.py`, otherwise the HAVE-FIM stays false after install, even if fim.so is there.

Reply:
I removed `tools` folder. None should be there.